component-runtime-beam/src/main/java/org/talend/sdk/component/runtime/beam/spi/record/AvroSchemaBuilder.java

```diff
@@ -232,11 +224,33 @@ public Schema build() {
             if (elementSchema == null) {
                 throw new IllegalStateException("No elementSchema set for this ARRAY schema");
             }
+            // FIXME: 7/12/21 => Here, it assumes that elementSchema inherit from Unwrappable<avro.Schema>
```

Comment:
if there's a potential fix, please create a jira issue and link it's preferable. thanks

Reply:
https://jira.talendforge.org/browse/TCOMP-1957